lib/rules/no-extra-semicolons/__tests__/index.js

```diff
@@ -555,6 +555,6 @@ testRule(rule, {
     code: "a { .mixin();\ncolor: red;; }",
     message: messages.rejected,
     line: 2,
-    column: 10,
+    column: 12,
```

Comment:
This should be even more correct, `12` corresponds to the last of the two semicolons.